spark-3.1-spanner-lib/src/main/java/com/google/cloud/spark/spanner/SpannerUtils.java

```diff
@@ -192,6 +194,18 @@ public static InternalRow spannerStructToInternalRow(
     return sparkRow;
   }
 
+  public String serializeMap(Map<String, String> m) throws Exception {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Why do we initialize an ObjectMapper on each call  of serialize deserialize?

Reply:
I wasn't sure whether the mapper was thread safe hence preferred to play it safe

Reply:
See this [SO question](https://stackoverflow.com/questions/57670466/objectmapper-best-practice-for-thread-safety-and-performance)

Reply:
Got it and thank you @davidrabinowitz I've fixed it up.